src/fluree/server/handler.clj

```diff
@@ -248,6 +248,15 @@
                   (handler req*))))
         (handler req)))))
 
+(def wrap-no-result
+  "Handle a nil response from a selectOne query with no results."
+  (fn [handler]
+    (fn [req]
+      (let [{:keys [status body] :as result} (handler req)]
+        (if (and (= 200 status) (nil? body))
```

Comment:
This feels like something that should be handled by muuntaja instead of rolling our own conversion for this specific case. Is there no way to accomplish this using that library since we're already using it to convert between formats? I can see this causing issues if we ever decide to respond with edn as well as json.

Reply:
I gave it an honest try with muuntaja but wasn't willing to devote more than 2 hours to it. Once I couldn't get it working in that window I went back to this. I figure that if we ever do support edn there will need to be a lot of updates to the middleware anyways.

Reply:
Is this a problem with the `QureryResponse` schema? What happens when you just account for the possibility that `:select-one` query responses can also be `nil`?

Reply:
Then it converts the `:body` key to the empty string `""`, which isn't valid json.